docs/explanation.rst

```diff
@@ -1,48 +1,155 @@
 
-Qolmat methods 
-==============
+Model Selection
+===============
 
-It can be divided into two main parts:
+Qolmat provides a convenient way to estimate optimal data imputation techniques by leveraging scikit-learn-compatible algorithms. Users can compare various methods based on different evaluation metrics.
 
-1. Impute missing values via multiple algorithms;
-2. Compare the imputation methods in a supervised manner.
+.. _general_approach:
 
-Imputation methods overview
----------------------------
+1. General approach
+-------------------
 
-For univariate time series:
+Let :math:`X \in \mathbb{R}^{n \times m}` be a dataset with missing data. We observe only the matrix :math:`X_{obs} = X \odot M \in (\mathbb{R} \cup \{NA\})^{n \times m}` with :math:`M` the mask such that :math:`M_{ij} = 1` if :math:`X_{ij}` is missing and 0 otherwise. Let :math:`f` be an imputation function which outputs a complete dataset based on the observed ones and the mask, i.e.
```

Comment:
Les produits d'Hadamard me semblent plus source de confusion. Peut-être simplement mettre X_obs = {X_{i,j} | (i,j) such that M_{i,j} = 1}, qui s'apparente à la fonction (i,j) -> X_{i,j} définie sur les i,j tels que M_ij=1.
De plus je ne comprends pas les deux flèches en série en dessous. Je dirais simplement qu'on cherche la fonction f définissant l'opération d'imputation par \hat X = f(X_obs), quitte à être un chouia moins précis... Et au lieu du crochet séparant les cas M=0 ou M=1 je préciserais que la fonction f doit vérifier f(\hat X_ij) = X_ij si M_ij=0